manifests/cloud-platform-alt-dns/coredns.yaml

```diff
@@ -0,0 +1,85 @@
+---
+kind: Pod
+apiVersion: v1
+metadata:
+  name: coredns
+  namespace: openshift-infra
```

Comment:
I don't think the namespace here actually matters (it only affects whether mirror pods show up in the API, which doesn't matter for bootstrap), but I would guess you want the new name instead.